Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
Good question!  I think that having a standard version would be possible.  My guess is that we'd need for the repos to point at the right `.jshintrc` file after the bower install.

Do we imagine that different repos might adjust the settings?  I'm wondering if putting the hint file in a https://github.com/edx/cookiecutter-django-ida would make sense or something like https://github.com/edx/edx-lint.